src/openloco/windows/CompanyWindow.cpp

```diff
@@ -0,0 +1,1986 @@
+                    self->y + self->widgets[widx::current_loan].top + 1,
+                    colour::black,
+                    string_ids::interest_per_year,
+                    _common_format_args);
+            }
+
+            // 'Cash' label with value
+            {
+                // Set cash value in format args.
+                *(uint32_t*)&_common_format_args[0] = company->cash.var_00;
+                *(uint16_t*)&_common_format_args[2] = company->cash.var_04;
+
+                string_id cash_format = string_ids::cash_positive;
+                if ((company->challenge_flags & company_flags::bankrupt) != 0)
+                    cash_format = string_ids::cash_bankrupt;
+                if (company->cash.var_04 < 0)
+                    cash_format = string_ids::cash_negative;
+
+                gfx::draw_string_494B3F(
+                    *dpi,
+                    self->x + 7,
+                    self->y + self->widgets[widx::current_loan].top + 13,
+                    colour::black,
+                    cash_format,
+                    _common_format_args);
+            }
+
+            // 'Company value' label with value
+            {
+                // Set company value in format args.
+                *(uint32_t*)&_common_format_args[0] = company->companyValue.var_00;
+                *(uint16_t*)&_common_format_args[2] = company->companyValue.var_04;
+
+                gfx::draw_string_494B3F(
+                    *dpi,
+                    self->x + 7,
+                    self->y + self->widgets[widx::current_loan].top + 26,
+                    colour::black,
+                    string_ids::company_value,
+                    _common_format_args);
+            }
+
+            // 'Profit from vehicles' label with value
+            {
+                // Set company value in format args.
+                *(uint32_t*)&_common_format_args[0] = company->vehicleProfit.var_00;
+                *(uint16_t*)&_common_format_args[2] = company->vehicleProfit.var_04;
+
+                gfx::draw_string_494B3F(
+                    *dpi,
+                    self->x + 7,
+                    self->y + self->widgets[widx::current_loan].top + 39,
+                    colour::black,
+                    string_ids::profit_from_vehicles,
+                    _common_format_args);
+            }
+        }
+
+        // 0x0043361E
+        static void draw_scroll(window* self, gfx::drawpixelinfo_t* context, uint32_t scrollIndex)
+        {
+            int16_t y = 47 - self->widgets[widx::scrollview].top + 14;
+
+            for (uint8_t i = 0; i < static_cast<uint8_t>(ExpenditureType::Count); i++)
+            {
+                // Add zebra stripes to even labels.
+                if (i % 2 == 0)
+                {
+                    auto colour = colour::get_shade(self->colours[1], 6) | 0x1000000;
+                    gfx::fill_rect(context, 0, y, 2176, y + 9, colour);
```

Comment:
```suggestion
                    gfx::fill_rect(context, 0, y, 128 * 17, y + 9, colour);
```
with `128` being the month width